port/unix/omrmmap.c

```diff
@@ -279,6 +279,23 @@ omrmmap_shutdown(struct OMRPortLibrary *portLibrary)
 int32_t
 omrmmap_startup(struct OMRPortLibrary *portLibrary)
 {
+#if defined(LINUX)
+	uintptr_t handle;
+	int rt = -1;
+	/* If omrsl_open_shared_library fails it already reports error, and we'll treat as if memfd_create is not avaiable */
+	if (0 == omrsl_open_shared_library(portLibrary, NULL, &handle, 0)) {
+		uintptr_t func = 0;
+		rt = omrsl_lookup_name(portLibrary, handle, "memfd_create", &func, NULL);
+		/* If omrsl_lookup_name does not succeed, func value remains 0 */
+		PPG_memfd_call = func;
+	}
+
+	if (0 != rt) {
+		/* If omrsl_lookup_name failed we'll just report and treat as if memfd_create is not avaiable */
+		portLibrary->error_set_last_error_with_message(portLibrary, OMRPORT_ERROR_VMEM_UNSUPPORTED_DOUBLE_MAP, "Double Map API lookup failed.");
+	}
```

Comment:
I don't think we need to set an error status here, since its not really an error and no one will check the result anyways.  I think this whole code block can be deleted.
```suggestion

```